### PR TITLE
Lint params

### PR DESCRIPTION
**Pull request by ArnoStiefvater**

### CODE REVIEW

**Review by jjnicola — APPROVED** [review]
I have not tested it. Maybe @cfi-gb can give it a try. Looks legit

File: nasl/lint.c [hunk 1]
```diff
@@ -216,6 +216,37 @@ nasl_lint_def (lex_ctxt *lexic, tree_cell *st, int lint_mode,
...
+                                            (GCompareFunc) list_cmp))
+                    func_params =
+                      g_slist_prepend (func_params, args->x.str_val);
+                  else
+                    {
+                      g_message ("%s: Error at or near line %d.",
+                                 finfo->caller_file, linenum);
+                      g_message ("Parameter \"%s\" passed to function \"%s\" "
+                                 "was provided multiple times.",
+                                 args->x.str_val, finfo->func_name);
```
Comment (cfi-gb) [inline nasl/lint.c:242]: Can we use a single message like e.g. the following please?

```suggestion
                      g_message ("%s: Error at or near line %d. "
                                 "Parameter \"%s\" passed to function \"%s\" "
                                 "was provided multiple times.",
                                 finfo->caller_file, linenum, args->x.str_val, finfo->func_name);
```

**Review by cfi-gb — CHANGES_REQUESTED** [review]

File: nasl/lint.c (hunk 1 above)
Comment (ArnoStiefvater) [reply nasl/lint.c]: done

**Review by cfi-gb — DISMISSED** [review]

**Review by cfi-gb — COMMENTED** [review]
Changes looks good:

```
lib  nasl-Message: 10:44:59.566: lint.nasl: Error at or near line 3. Parameter "test_version" passed to function "version_in_range" was provided multiple times.
Error while processing lint.nasl.
1 errors found
```

I have also done a full run of openvas-nasl-lint to check for regressions / other problems and haven't found any.

Finally code review and approval needs to be done by a @greenbone/gvm-dev member.

**Review by jjnicola — APPROVED** [review]

**Pull request merged**